buildTest.sh

```diff
@@ -6,21 +6,30 @@ NC='\033[0m' # No Color
 
 echo "Checking for required files..."
 echo -ne "  UI index: "
-if [ ! -f ./ui/dist/index.js ]; then echo -e "${RED}Not Found${NC}" && exit 1; else echo -e "${GREEN}Found${NC}"; fi
+if [ ! -f ./ui/dist/ui/src/index.js ]; then echo -e "${RED}Not Found${NC}" && exit 1; else echo -e "${GREEN}Found${NC}"; fi
```

Comment:
we should not be changing the path where the compiled components go...the final results should be in /dist, not two folders below that. There shouldn't be a /src folder under a /dist folder.